src/screens/privacy-policy/privacy-policy.tsx

```diff
@@ -1,27 +1,51 @@
 import React from 'react';
-import { View } from 'react-native';
-import { NavigationParams, NavigationScreenProp, NavigationState } from 'react-navigation';
-
-import stylesProvider from './styles';
-import { withTheme } from '../../core/theme/with-theme';
-import { Text } from '../../library';
+import { Image, View } from 'react-native';
+import { withNavigationParams, INavigationProps } from '../../navigation/with-navigation-params';
+import { withTheme, IThemeProps } from '../../core/theme/with-theme';
+import { IReduxState } from '../../redux/state';
 import { translate } from '../../core/i18n';
+import { Text } from '../../library';
 
-export interface IProps {
-    navigation: NavigationScreenProp<NavigationState, NavigationParams>;
-    styles: ReturnType<typeof stylesProvider>;
-}
+import stylesProvider from './styles';
+import { smartConnect } from '../../core/utils/smart-connect';
+import { connect } from 'react-redux';
+import { themes } from '../../navigation/navigation';
 
-export const PrivacyPolicyScreenComponent = (props: IProps) => (
-    <View style={props.styles.container}>
-        <Text>Privacy policy scroll</Text>
-    </View>
-);
+export const mapStateToProps = (state: IReduxState) => {
```

Comment:
I think that this screen will be dummy, it won't need redux, you can remove redux stuff from here